lms/static/sass/views/_teams.scss

```diff
@@ -148,6 +149,200 @@
     @extend %t-weight4;
   }
 
+  .search-tools {
+    @extend %t-copy-sub1;
+    margin: ($baseline/10) $baseline;
+    color: $gray-l1;
+
+    p, span { // override
+      color: inherit;
+    }
+  }
+
+  .card {
+    @extend .clearfix;
+    position: relative;
+    display: inline-block;
+    width: 30%;
+    margin: ($baseline) 1.2%;
```

Comment:
this is a flexible margin to let the layout flex a bit - i would like to use susy, and neat (which I'm not keen on anyway) is not available in this space, so I went with strict percentages for now. Should I mark them so later cleanup is easy?